go-controller/pkg/ovn/pods.go

```diff
@@ -665,15 +665,9 @@ func (oc *Controller) getPortAddresses(nodeName, portName string) (net.HardwareA
 	return podMac, podIPNets, nil
 }
 
-// ovnNBLSPDel deletes the given logical switch using the libovsdb library
-func (oc *Controller) ovnNBLSPDel(logicalPort, logicalSwitch, lspUUID string) ([]ovsdb.Operation, error) {
+// delLSPOps returns the operations required to delete the given logical switch using the libovsdb library
```

Comment:
What stand for delLSPOps? Logical Switch "P"ort?   delLogicalSwitchPort()  or deleteLogicalSwitchPort() ?  Easier to read IMHO.

Reply:
The doc string is wrong. Will fix it in the next iteration. We already have another delLogicalSwitchPort function. I’ll probably make it explicit in the doc string.

Reply:
:100: :1st_place_medal: